python/lsst/pipe/tasks/multiBand.py

```diff
@@ -282,7 +285,7 @@ def runDataRef(self, patchRef):
             exposure = patchRef.get("fakes_" + self.config.coaddName + "Coadd", immediate=True)
         else:
             exposure = patchRef.get(self.config.coaddName + "Coadd", immediate=True)
-        expId = int(patchRef.get(self.config.coaddName + "CoaddId"))
+        expId = getGen3CoaddExposureId(patchRef, coaddName=self.config.coaddName, log=self.log)
```

Comment:
So why do those other places you use `getGen3CoaddExposure` need a try block and not this one?

Reply:
Ah, sorry...I added a try block in the [getGen3CoaddExposureId function](https://github.com/lsst/coadd_utils/pull/30/commits/f09958e309f6e32fd1fff143f57b0c88d231616c#diff-c4961a7427d6736ea2e9052d39925f90d1ef7b2e0c7175fee9cc60c25b89f7bcR79-R85) and forgot to remove all of the now-redundant ones.  I tried to give Jim all the id wrangling bits, but some repos had a mixture...